src/main/java/by/exadel/internship/team_four/entity/enums/FormStatus.java

```diff
@@ -0,0 +1,18 @@
+package by.exadel.internship.team_four.entity.enums;
+
+public enum FormStatus {
+    REGISTERED,
+    ADMIN_INTERVIEW_ASSIGNED,
+    ADMIN_INTERVIEW_PASSED,
+    TECH_INTERVIEW_ASSIGNED,
+    TECH_INTERVIEW_PASSED,
+    ACCEPTED,
+    NOT_MATCHED,      //не подходит (example нет офиса, не тот регион, и т д английский) нужен ли???
+    REJECTED,         // отклонен (не соответствие требованиям (тех знания))
+    CANDIDATE_REFUSED,   // отказался (сам кандидат)
+
+    WAITING, //спросить
```

Comment:
Save these statuses to some documentation and remove now till real implementation

Reply:
is it good practice, if we dont use it now, we should keep them to some documentation? because we will use them for first version our project (second resources)? it is not for future versions

Reply:
As far as I know business logic implementation  is postponed for next statuses WAITING, GOT_JOB_OFFER and DIDNT_GOT_JOB_OFFER. I said only about these three statuses above.

Reply:
done